shopinvader/models/product_product.py

```diff
@@ -59,13 +59,24 @@ def _add_to_cart_allowed(self, backend, partner=None):
             self.shopinvader_bind_ids.filtered(lambda x: x.backend_id == backend)
         )
 
-    def _get_invader_variant(self, backend, lang):
+    def _get_invader_variant(self, backend, lang, only_active=True):
         """Retrieve invader variant by backend and language.
 
         :param backend: backend recordset
         :param lang: lang code
         """
-        return self.shopinvader_bind_ids.filtered(
+        # We perform a first check to include inactive bindings
+        # if either the "only_active" argument is False or
+        # the "active_test" flag from the context is not found.
+        bindings = self.shopinvader_bind_ids
+        active_test = None
+        if "active_test" in self.env.context:
```

Comment:
I would expect to see a change here
https://github.com/shopinvader/odoo-shopinvader/blob/6898c93dcc1af3419c4a3b517a4abb03492a5593/shopinvader_wishlist/models/product_set.py#L52
you need to remove that line and call "_get_invader_variant" with explicit args "only_active=False"
and in "_get_invader_variant" method do not check what is in context just set the context according to the only_active

also, you may check other calls _get_invader_variant, if the context is used there we also want to replace it on flag  usage